VespaGAS/ViewModels/GasCalculatorViewModel.swift

```diff
@@ -1,23 +1,41 @@
 import Foundation
 import RxSwift
+import ReSwift
 
-struct GasCalculatorViewModel {
-    let oilMixValueObservable: Observable<String>
-    let oilValueObservable: Observable<String>
-    let gasValueObservable: Observable<String>
+class GasCalculatorViewModel: StoreSubscriber {
+    private let oilMixBehaviourSubject = BehaviorSubject(value: OilMix(amount: 0).description)
```

Comment:
Here I had to change my ViewModel to be a class in order to have a **deinit** that I use to unregister to the store updates. Is there a better way to do it?